src/sentry/issues/ongoing.py

```diff
@@ -31,11 +31,13 @@ def bulk_transition_group_to_ongoing(
             activity_type=ActivityType.AUTO_SET_ONGOING,
             activity_data=activity_data,
             send_activity_notification=False,
+            from_substatus=from_substatus,
         )
 
     for group in groups_to_transistion:
         group.status = GroupStatus.UNRESOLVED
         group.substatus = GroupSubStatus.ONGOING
+        # TODO(snigdha): do we need to update the priority here?
```

Comment:
@malwilley do we want to update the priority here? not sure if we want to include it in the post-save send below

Reply:
I don't think it's necessary yet but may be in the future when we want to sync `priority` to Snuba